### PR TITLE
Fix mounting SSL files from secret into Pod as files to given sslDir mountpoint

### PR DESCRIPTION
**Pull request by reixd**

### CODE REVIEW

File: dysnix/proxysql/templates/statefullset.yaml [hunk 1]
```diff
@@ -74,10 +74,26 @@ spec:
...
+            {{- if .Values.ssl.cert }}
+            - name: secrets
+              mountPath: {{ include "proxysql.sslDir" . }}/{{ .Values.ssl.cert_file }}
+              subPath: cert.pem
+            {{- end }}
+            {{- if .Values.ssl.key }}
+            - name: secrets
+              mountPath: {{ include "proxysql.sslDir" . }}/{{ .Values.ssl.key_file }}
+              subPath: key.pem
+            {{- end }}
```
Comment (dennybaa) [inline dysnix/proxysql/templates/statefullset.yaml:92]: Hello @reixd. Could you please explain why do we mount same secret 3 times?

Comment (reixd) [reply dysnix/proxysql/templates/statefullset.yaml:92]: yeah, sure. 
This is not the same secret 3 times, but instead the required SSL files each one mounted separately. And possibly in different places in the file system, than the defaults.
When the SSL files (ca, cert, key) are inline stored from the chart values in the secret resource, I need to tell k8s which key:value pair from the secret is what. 
I have this PR running in my cluster and it works.

Comment (reixd) [reply dysnix/proxysql/templates/statefullset.yaml:92]: And each pod (deployment, daemonset, statefullset) requires the SSL files.

**Comment by reixd** [comment]
@dennybaa do you have any comments on this PR?

File: dysnix/proxysql/templates/statefullset.yaml (hunk 1 above)
Comment (reixd) [reply dysnix/proxysql/templates/statefullset.yaml:92]: Yes, selectable names and path are required.

> If you have a real-world scenario and need this, np let's go on and merge. Though I would prefer fixed names in favor of less lines of code and simplicity :)

I do have exactly this use case: 
* The proxysql core server is an already existing (legacy) setup
* The ssl certs are stored as `/etc/mysql/ssl/ca.pem`, `/etc/mysql/ssl/client.pem`, `/etc/mysql/ssl/client.key`
* The newer kubernetes setup can use any other paths, however due to the clustering feature of proxysql, *ALL* nodes (core+satellite) share the *same configuration*.
* After the proxysql satellite nodes (@ k8s) sync the config from the core nodes, the satellite nodes try to use the SSL certs from `/etc/mysql/ssl/` and hence the need to specify a mount path and filename.

I hope it is clear enough. Tried to minimize extreme customization and keep sane defaults.

Comment (dennybaa) [reply dysnix/proxysql/templates/statefullset.yaml:92]: okay sounds good. Thank you @reixd

**Pull request merged**